python-checks/src/test/resources/checks/argumentType.py

```diff
@@ -59,25 +59,27 @@ def stdlib_functions():
   A = UnexpectedClass()
   acos(A) # Noncompliant
   B = datetime.tzinfo()
-  B.tzname(42) # Noncompliant {{Change this argument; Function "tzname" expects a different type}}
-#          ^^
+  B.tzname(42) # FN Noncompliant {{Change this argument; Function "tzname" expects a different type}}
```

Comment:
We seem to resolve that the method is originally an abstract method (has the `abstractmethod` decorator), which we consider unknown, and we bail because of that.